annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -0,0 +1,6 @@
+spring.batch.job.enabled=false
+genomenexus.base_url=
+genomenexus.hgvs=variant_annotation/hgvs/
+genomenexus.isoform_override=variant_annotation/isoform_override/
```

Comment:
Should this be a runtime argument?  Do we want to have to repackage/launch the pipeline if we want to change isoform files (say between MSK and Uniprot?)